docs/source/dataframe-create.rst

```diff
@@ -33,6 +33,7 @@ File Formats:
     read_fwf
     from_bcolz
     from_array
+    from_dict
```

Comment:
Just a thought, maybe `from_dict` might be better suited under a different heading "Python Collections"?

This works as-is too because `from_array` also doesn't perfectly fit under "file formats". I'll leave it to you. :)

Reply:
@pavithraes - this is a good point.  I'm actually not entirely sure how `from_array` works, so not sure the best categorization.  I'm guessing it's not referring to a Dask Array cause there is a separate `from_dask_array` method.  I guess I'd argue to keep this as-is, mainly cause I don't know enough to have an opinion.

Reply:
Sounds good!